frontend/style/main.css

```
@@ -1,44 +1,78 @@
+  --color-text-secondary: #7a7a85;
+  --color-text-tertiary: #4a4a55;
+  --color-accent: #a78bfa;
+  --color-accent-hover: #8b6ff0;
+  --color-accent-muted: rgba(167, 139, 250, 0.12);
+  --color-border: #222228;
+  --color-border-subtle: #1a1a1f;
+  --color-error: #ef4444;
+  --color-error-bg: rgba(127, 29, 29, 0.4);
+
+  /* Typography — system font stack */
+  --font-sans: system-ui, -apple-system, "Segoe UI", Roboto, Helvetica, Arial, sans-serif;
```

Stylelint (17.9.0)

[error] Expected "Roboto" to be "roboto" (value-keyword-case)

(value-keyword-case)